src/components/StatueListContainer.js

```diff
@@ -0,0 +1,23 @@
+import React, { Component } from "react";
+import StatueList from "./Statuelist";
+import SlaverStatues from "../Statues/slavers";
+
+export class StatueListContainer extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      statues: SlaverStatues,
+    };
+  }
+
+  render() {
+    return (
+      <div>
+        <h1>Statues</h1>
```

Comment:
Worth creating a labels file for this.

Reply:
We are a bit confused by what you mean exactly - Could you explain it a bit more ...